portal/models/i18n_utils.py

```diff
@@ -201,11 +201,15 @@ def pos_from_zip(zipfile):
         yield locale_code, po
 
 
-def msgcat(*po_files):
-    """Concatenate input po_files together, with later files overwriting earlier ones"""
+def msgcat(*po_files, **kwargs):
+    """
```

Comment:
when you have a single line comment - it should just be a single line, not a block, i.e.
`"""Concatenate input po_files together, with later files overwriting earlier ones"""`